src/powerquery-language-services/currentDocumentSymbolProvider.ts

```diff
@@ -41,6 +41,7 @@ export class CurrentDocumentSymbolProvider implements SymbolProvider {
         ) {
             return [];
         }
-        return InspectionUtils.getSymbolsForInspectionScope(this.maybeTriedInspection.value);
+
+        return [...InspectionUtils.getSymbolsForInspectionScope(this.maybeTriedInspection.value.triedNodeScope)];
```

Comment:
what does this change?

Reply:
Actually, that should be tweaked to not have the spread operator. I originally was experimenting with something else there that was more than nit, but it looks like I didn't fully revert it.

The actual nit is to change `this.maybeTriedInspection.value` -> `this.maybeTriedInspection.value.**triedNodeScope**` which has stricter typing.